CONTRIBUTING.md

```diff
@@ -16,13 +16,21 @@ Don't hesitate to [contact](https://github.com/easafe) if you need help, or want
 
 MeroChat is written in PureScript, both server-side and client-side. PostgreSQL is used for persistance.
 
+
+### Containers
+
 You can run MeroChat locally with podman or docker.
 
-1. Run `sudo docker-compose -f containers/composefile.yaml up` to spin up the containers
+Just run `podman-compose -f containers/composefile.yaml up` to spin up the containers.
+Your very own MeroChat is now accessible at http://localhost:8000, and will be rebuilt when you make changes to the code. This isn't instantaneous though. Keep an eye at the container runtime output.
+
+Finally tear the containers down with `ctr+c` followed by `podman-compose -f containers/composefile.yaml down`
```

Comment:
Should we mention docker here too?

Reply:
We can do that yes, though I  hold that podman is better 😏

Reply:
you are the boss